jiant/scripts/download_data/constants.py

```diff
@@ -24,6 +24,20 @@
     "piqa",
     "winogrande",
     "ropes",
+    "acceptability_definiteness",
+    "acceptability_coord",
+    "acceptability_eos",
+    "acceptability_whwords",
+    "senteval_bigram_shift",
+    "senteval_coordination_inversion",
+    "senteval_obj_number",
+    "senteval_odd_man_out",
+    "senteval_past_present",
+    "senteval_sentence_length",
+    "senteval_subj_number",
+    "senteval_top_constituents",
+    "senteval_tree_depth",
+    "senteval_word_content",
```

Comment:
The number of tasks here differs from the added `supported_tasks.md`. Are there some missing?

Reply:
The ANLI row got included in the diff because I also added the checkmark for being downloadable. There should only be 4 new acceptability tasks and 10 senteval tasks.